### PR TITLE
Fix/oatsd 2213/fix taskqueue memory issue

### PR DESCRIPTION
**Pull request by siwane**
**Description**
With 800k tasks in `tq_task_log`,  taskqueue monitoring is causes memory error.
By removing php abstraction TaskLogEntity and by querying directly for timeout, php process is smoother

This PR is only about changing interfaces. To have all required pieces please test with https://github.com/oat-sa/tao-core/pull/3685

**Ticket**
https://oat-sa.atlassian.net/browse/OATSD-2213

**How to test**
Generate some tasks for last month. 
Launch taoSystemStatus page, taskqueue monitoring
=> Taskqueue graph is displayed as usual
To go deeper generate 100k tasks

### CODE REVIEW

File: model/Check/System/TaskQueueFinishedCheck.php [hunk 1]
```diff
@@ -109,48 +115,41 @@ private function getTasksStatistics() : array
 
         $result = [];
         foreach (self::OPTIONS_INTERVAL as $name => $interval) {
-            $intervalObj = new \DateInterval($interval);
-            $period = new \DateInterval($name);
+            $intervalObj = new DateInterval($interval);
+            $period = new DateInterval($name);
 
             $points = round($this->getIntervalSeconds($period) / $this->getIntervalSeconds($intervalObj));
```
Comment (uncleempty) [inline model/Check/System/TaskQueueFinishedCheck.php]: I would ask you to split this line into 2 or 3 lines, with an `if` block. Reason: even if most of the code below was changed, the function called in this line was not - there is still space for `division by zero`.

**Review by uncleempty — CHANGES_REQUESTED** [review]
:+1: , just one small change to add

File: model/Check/System/TaskQueueFinishedCheck.php (hunk 1 above)
Comment (siwane) [reply model/Check/System/TaskQueueFinishedCheck.php]: done

**Review by uncleempty — APPROVED** [review]
:+1:

**Pull request merged**